notebooks/official/pipelines/pipelines_intro_kfp.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>delete this cell. Doesn't seem to make sense or belong here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/494/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>